### ISSUE TITLE
Error: unmatshal Download

### ISSUE THREAD

**Issue by njzjz** [Mikubill/cowtransfer-uploader#10]
```
cowtransfer-uploader https://c-t.work/s/7eb3eacb52c546
Remote: https://c-t.work/s/7eb3eacb52c546
Error: unmatshal DownloadDetails returns error: unexpected end of JSON input
```

**Comment by njzjz** [comment]
和#8 的错误差不多

**Comment by Mikubill** [comment]
fixed (v0.4.8 https://github.com/Mikubill/cowtransfer-uploader/releases/tag/v0.4.8)
